include/tsv/tsv.hpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2015 Adrien Devresse <adrien.devresse@epfl.ch>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#pragma once
+
+#ifndef H5_USE_BOOST
+#define H5_USE_BOOST
+#endif
+
+#include <string>
+#include <vector>
+#include <unordered_map>
+
+#include <boost/integer.hpp>
+
+namespace TSV {
+
+///
+/// \brief The TSVInfo class
+///
+/// Includes all the information of a mecombo entry of the tsv file
+///
+class TSVInfo {
+public:
+    std::string morphName;
+    boost::int32_t layer;
+    std::string fullMType;
+    std::string eType;
+    std::string eModel;
+    std::string comboName;
+    double thresholdCurrent;
+    double holdingCurrent;
+};
+
+struct pair_hash
+{
+    template <class T1, class T2>
+    std::size_t operator() (const std::pair<T1, T2> &pair) const
+    {
+        return std::hash<T1>()(pair.first) ^ std::hash<T2>()(pair.second);
+    }
+};
+
+typedef std::unordered_map<std::pair<std::string, std::string>, TSVInfo, pair_hash> unordered_pair_map;
```

Comment:
`using unordered_pair_map = …`

Reply:
historical note: Only in C++14 stdlib moved to "using" since it can be templated :)
I think we can follow the same. If we can bump to c++14 (we can right??) then I'd also move to `using`

Reply:
From a quick search I found out that `using` was introduced from `C++11`, so we can still use it